docs/roadmap.rst

```diff
@@ -0,0 +1,102 @@
+.. _roadmap:
+
+Project's Roadmap
+=================
+
+``skops`` is a project which deals with certain challenges related to
+operationalizing scikit-learn based models. To that end, the following areas
+are in our focus, and are already implemented or will be in worked on in the
+near future.
+
+Model Cards
+-----------
+
+We have now :ref:`tools <model_card>` to generate model cards, but there
+are still some rough edges which need to be smoothed out. Model cards have
+the potential to be adopted by upstream scikit-learn, which would be
+possible since we have almost no external dependency there. You can see the
+list of open issues on model cards `here
+<https://github.com/skops-dev/skops/issues?q=is%3Aissue+is%3Aopen+label%3A%22model+cards%22+>`__.
+In particular:
+
+- An app to create or modify a model card: This would also include easy
+  ways for users to add inspection of the models to the card. They can
+  potentially upload the model and the data, and easily generate good
+  visualizations about different aspects of the model. This work is started
+  in :pr:`307`.
+- There are issues with model cards that make it a bit tricky to work with
+  them, such as linking images. These issues need to be resolves so that
+  new users don't encounter too many hurdles when they first start using
+  the tools.
+
+Persistence
+-----------
+When deploying models, persistence is a key aspect. Since :ref:`pickles are
+insecure <persistence>`, we helps users in two ways to replace pickle
+files. One is our own ``.skops`` format, and another one is through tools
+to make it easier to convert models to ONNX. There are existing tools to
```

Comment:
Should probably be "sklearn models"?

Reply:
Ideally they'd be able to convert non-sklearn models as well, by implementing custom converters. But that's not well documented.